node/metrics/snarkOS-grafana.json

```diff
@@ -203,7 +203,7 @@
             "uid": "${DS_PROMETHEUS}"
           },
           "editorMode": "code",
-          "expr": "avg(\n  1 / rate(snarkos_blocks_height_total{}[1m])\n) < +inf",
+          "expr": "avg(\n  1 / rate(snarkos_bft_height_total{}[1m])\n) < +inf",
```

Comment:
Changed these names just to reflect changes in https://github.com/AleoHQ/snarkOS/pull/3210